spec/rspec/core/profiler_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rspec/core/profiler'
+
+RSpec.describe 'RSpec::Core::Profiler' do
+  let(:profiler) { RSpec::Core::Profiler.new }
+
+  it 'has starts with an empty hash of example_groups' do
+    expect(profiler.example_groups).to be_empty.and be_a Hash
+  end
+
+  context 'when hooked into the reporter' do
+    include FormatterSupport
+
+    let(:id) { 42 }
+    let(:description ) { "My Group" }
+    let(:now) { ::Time.now }
```

Comment:
In my experience, when you don't care what the time used is it's best to specify it as an explicit static value (e.g. `Time.utc(2015, 5, 28, 12, 30)`).  Otherwise you're prone to having failures when leap seconds or DST transitions occur while running the suite.